### PR TITLE
Fixed is_conversation_view() not working in new gmail UI

### PR DESCRIPTION
**Pull request by shashikiran-im**
Fixed the issue of the method is_conversation_view() not working in new gmail UI. Now it works in both classic and new gmail interface.

### CODE REVIEW

**Comment by josteink** [comment]
Just to have the actual issue linked to PR, this PR is meant to fix https://github.com/KartikTalwar/gmail.js/issues/485.

**Comment by josteink** [comment]
LGTM. Thanks for the patch!

I've bumped the version and published a new package with these changes: 0.7.4.

File: src/gmail.js [hunk 1]
```diff
@@ -2567,25 +2567,25 @@ var Gmail = function(localJQuery) {
...
-        var array_with_flag = api.tracker.globals[17][4][1];
-
-        for (var i = 0; i < array_with_flag.length; i++) {
-            var current = array_with_flag[i];
-
-            if (current[0] === flag_name) {
-                flag_value = current[1];
-
-                break;
+        if( window.gmail.check.is_new_gui() ) {
```
Comment (josteink) [inline src/gmail.js]: Oh. This line is actually *wrong*. My bad for not correcting it.

I will fix it tomorrow and issue an update as 0.7.5.

Comment (thinksysprem) [reply src/gmail.js]: I have created a small PR #496 for this. I have tested it locally. If you want you can review, approve and release it. 

Thanks

**Pull request merged**